packages/ui-concerto/src/lib/formgenerator.js

```diff
@@ -76,10 +76,13 @@ class FormGenerator {
    */
   async loadFromText(texts) {
     this.loaded = false;
-
     this.modelManager.clearModelFiles();
     try {
-      texts.forEach(text => this.modelManager.addModelFile(text, null, true));
+      texts.forEach(text => {
+        // Unescape URL encoded relationship symbols
```

Comment:
I don't understand this. How did the model files get encoded?

Reply:
This appears to be related to use of storyboard. When I change the values in the knob editor, I see these encoded values.